### ISSUE TITLE
vApp 'description' not working

### ISSUE THREAD

**Issue by kokkerhout** [vmware/terraform-provider-vcd#633]
### Terraform Version
Terraform v0.14.5

### Affected Resource(s)
- vcd_vapp

### Terraform Configuration Files
```hcl
terraform {
  required_providers {
    vcd = {
      source  = "vmware/vcd"
      version = ">=3.1.0"
    }
  }
  required_version = ">= 0.14.2"
}

resource "vcd_vapp" "vapps" {
  for_each = var.vapps

  name                 = each.key
  org                  = each.value.org
  vdc                  = each.value.vdc_name
  description          = each.value.description

}

```

### Debug Output
https://gist.github.com/kokkerhout/c1987cf94472b913ab59f22a2991b64b

### Expected Behavior
When I define "description" it should add the description in the vApp

### Actual Behavior
The 'description' option in the module 'vcd_vapp' doesn't do anything. So everytime I run terraform apply it tells me it wants to add a description for the vApp because it fails to do it everytime. 

### Steps to Reproduce
1. `Add value for 'description'`
2. `terraform apply`
3. `Message it wants to add description`
3. `apply completes but no description in the dashboard`
4. `terraform apply`
5. `Message it wants to add description`
6. `apply completes but no description in the dashboard`

**Comment by Didainius** [comment]
Hello,
I have tried two things:
* Add a description during initial "create" phase
* Add a description later on (apply an update).

Here is my example HCL (excerpt, because other things don't really matter here):

```hcl

resource "vcd_vapp_org_network" "tf-net" {
  vapp_name         = vcd_vapp.web.name
  org_network_name  = vcd_network_isolated.tf-net.name
  
  depends_on = [
    vcd_vapp.web,
  ]
}

resource "vcd_vapp_vm" "web" {
  count = 5

  vapp_name     = vcd_vapp.web.name
  name          = "web1-${count.index}"
  description = "VM-${count.index}"

  catalog_name  = "cat-dainius"
  template_name = "photon-hw11"

  memory        = 2048
  cpus          = 1

  network {
    type               = "org"
    name               = vcd_vapp_org_network.tf-net.org_network_name
    ip_allocation_mode = "POOL"
  }

  depends_on = [
    # vcd_catalog_item.myNewCatalogItem,
    vcd_vapp.web,
  ]
}
```

Here is the output when I skipped `description` initially, but then added it later (because it inherited it from VM template initially):
```shell
 vcd_vapp_vm.web[0] will be updated in-place
  ~ resource "vcd_vapp_vm" "web" {
      ~ description                    = <<-EOT
          - This OVA provides a minimal installed profile of PhotonOS.
          - 
          -    Default password for root user is changeme. However user will be prompted to change the password during first login.
          - 
          -         
          + VM-0
        EOT
        id                             = "urn:vcloud:vm:01eef68d-1607-420f-a44c-bb19c5f4791a"
        name                           = "web1-0"
        # (21 unchanged attributes hidden)


        # (2 unchanged blocks hidden)
    }
```

And here is the result after that:
<img width="749" alt="image" src="https://user-images.githubusercontent.com/15804230/106507649-04204480-64d4-11eb-9e67-17c94ca54839.png">




* Here is also the description set during "initial" create phase:

<img width="614" alt="image" src="https://user-images.githubusercontent.com/15804230/106508369-fd460180-64d4-11eb-8e2c-a3de5e676b3b.png">

May I ask you to enable SDK logs by using https://registry.terraform.io/providers/vmware/vcd/latest/docs#logging additionally and share it. That would allow to dig even further into the issue and show if the API calls contain description

Thanks!

**Comment by Didainius** [comment]
@kokkerhout ,
One more thought. Can this be because of `vcd_vm_internal_disk` usage which is described in issue #630

**Comment by kokkerhout** [comment]
Hi, you tried to add a description to a VM. This bug is about a description of the vApp.
Indeed I have no issues adding a description to the VM. Issue occurs at the vApp -> module 'vcd_vapp'

**Comment by CarloPrinzi** [comment]
I've run into this same problem, "description" is also missing from the documentation for "vcd_vapp", so it looks like it's just been completely overlooked - I would really like to populate the vApp description field through terraform for something I'm working on - please implement this!

**Comment by Didainius** [comment]
@dataclouder fixed this issue in #664
